src/Domain/AclGroup.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Domain;
+
+class AclGroup
+{
+    private ?int $id;
```

Comment:
Property must not be accessed before initialization
```suggestion
    private ?int $id = null;
```